pyLODE fail, GitHub action.  New version rdfLib at fault?

I have a [github action](https://github.com/Loop3D/GKM/actions/workflows/updateDocs.yml) set up to run pyLODE on the ttl in my [github repo](https://github.com/Loop3D/GKM). Last week it worked; when I did a push today I got an error on the action. looking at the logs, the difference seems to be this:
### works (last week):
...
> Collecting rdflib>=5.0.0
>   Downloading **rdflib-5.0.0**-py3-none-any.whl (231 kB)

...

> Successfully installed isodate-0.6.0 markdown-3.3.4 pylode-2.10.3 **rdflib-5.0.0** rdflib-jsonld-0.5.0

... starts generating html files

### Doesn't work (today)
...
>Collecting rdflib>=5.0.0
>  Downloading **rdflib-6.0.0**-py3-none-any.whl (376 kB)

...

> Successfully installed isodate-0.6.0 markdown-3.3.4 pylode-2.10.3 **rdflib-6.0.0** rdflib-jsonld-0.5.0

then:

>/usr/local/lib/python3.8/dist-packages/pylode/profiles/ontdoc.py:763: UserWarning: Code: domainIncludes is not defined in  namespace SDO
>   for o in self.G.objects(subject=s, predicate=SDO.domainIncludes):
> /usr/local/lib/python3.8/dist-packages/pylode/profiles/ontdoc.py:819: UserWarning: Code: rangeIncludes is not defined in namespace SDO
>   for o in self.G.objects(subject=s, predicate=SDO.rangeIncludes):

.... stack trace omitted...

> File "/usr/local/lib/python3.8/dist-packages/pylode/profiles/base.py", line 467, in _make_schemaorg_metadata
>     return g.serialize(format="json-ld").decode("utf-8")
> AttributeError: 'str' object has no attribute 'decode'
> Error: Process completed with exit code 1.

any ideas?

--- a/pylode/profiles/base.py
+++ b/pylode/profiles/base.py
@@ -464,7 +464,7 @@ class BaseProfile:
         from rdflib.plugin import register, Serializer
         register('json-ld', Serializer, 'rdflib_jsonld.serializer', 'JsonLDSerializer')
 
-        return g.serialize(format="json-ld").decode("utf-8")
+        return g.serialize(format="json-ld", encoding="utf-8")  # support >= rdflib 6.0.0 and ensure backwards compat (last python 2 release)
 
     def _make_agent_link(self, name, url=None, email=None, affiliation=None):
         if self.outputformat == "md":

